packages/utils/src/formattingSpace.ts

```diff
@@ -127,7 +148,15 @@ function _isBlockEdge(node: Node | Node, side: 'start' | 'end'): boolean {
     // sibling on the specified side
     const siblingSide = side === 'start' ? 'previousSibling' : 'nextSibling';
     return ancestorsUpToBlock.every(ancestor => {
-        return !ancestor[siblingSide];
+        let siblingNode = ancestor[siblingSide];
```

Comment:
Again a detail but I think `sibling` would have done just fine as a variable name. Given that we use Typescript we generally don't put the type of the variable in its name.